src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/VisitSupport.kt

```diff
@@ -26,15 +26,12 @@ class VisitSupport(
   val id: Long = 0,
 
   @Column(name = "VISIT_ID", nullable = false)
-  var visitId: Long,
+  val visitId: Long,
 
-  @Column(name = "TYPE", nullable = false)
-  var type: String,
+  @Column(name = "DESCRIPTION", nullable = false)
+  var description: String? = null,
```

Comment:
nullable is false.

Reply:
Yes this is correct if there is a visit support it must have a description